terraform/channels.tf

```diff
@@ -229,11 +229,24 @@ resource "fastly_service_vcl" "channels" {
       # Note: we need to match on 301s and 302s here, since Fastly has multiple
       # layers, and otherwise a redirect might still get cached at the second
       # layer after the first layer turned a 301 into a 302.
+      #
+      # Additionally, this also implements the "Lockable HTTP Tarball Protocol"
+      # to use nixexprs.tar.xz with Flakes and have it locked properly.
```

Comment:
@emilylange do you still want to apply the changes you mentioned in matrix, or is this good as is?

Reply:
good as is :)